packages/hub/node-tests/routes/email-card-drop-request-test.ts

```diff
@@ -170,6 +191,21 @@ describe('POST /api/email-card-drop-requests', function () {
   });
 
   it('persists an email card drop request and triggers jobs', async function () {
+    let emailCardDropRequestsQueries = await getContainer().lookup('email-card-drop-requests', { type: 'query' });
+    let insertionTimeBeyondExpiry = Date.now() - emailVerificationLinkExpiryMinutes * 2 * 60 * 1000;
```

Comment:
Should this be relative to `fakeTime`?

Reply:
I was using that before but because the SQL query uses a non-fake `now()` it didn’t work… [mocking `now()` exists apparently](https://gist.github.com/thehesiod/d0314d599c721216f075375c667e2d9a) but seems pretty dramatique 😳